resources/profiles/Prusa.json

```diff
@@ -19,6 +19,14 @@
         {
             "name": "MINI",
             "sub_path": "machine/Prusa MINI.json"
+        },
+        {
+            "name": "XL",
```

Comment:
Shall we call it "Prusa XL" here?
XL feels a bit too short :)

Reply:
Agreed! This has now been fixed.